### PR TITLE
Feature/us4441 upgrade cms directive to ng2

### PR DESCRIPTION
**Pull request by richardbarker**

### CODE REVIEW

File: crossroads.net/core/services/contentMessage.service.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Rx';
+//import { Observable } from 'rxjs/Observable'
```
Comment (danrye) [inline crossroads.net/core/services/contentMessage.service.ts]: remove commented out code.

File: crossroads.net/app/downgrades.ts [hunk 2]
```diff
@@ -12,4 +14,11 @@ angular.module('crossroads')
     .directive('streaming', upgradeAdapter.downgradeNg2Component(StreamingComponent));
 
 angular.module('crossroads')
+    .directive('dynamic-content-ng2', upgradeAdapter.downgradeNg2Component(DynamicContentNg2Component));
```
Comment (silbermm) [inline crossroads.net/app/downgrades.ts]: Can't we chain all of these directives together? i.e

```
angular.module('crossroads')
    .directive('ng2Test', upgradeAdapter.downgradeNg2Component(Ng2TestComponent))
    .directive('streaming', upgradeAdapter.downgradeNg2Component(StreamingComponent))
    .directive('dynamic-content-ng2', upgradeAdapter.downgradeNg2Component(DynamicContentNg2Component))
    .directive('streamingVideo', upgradeAdapter.downgradeNg2Component(VideoComponent))

 ;
```

Not a necessity, but it is what we are doing everywhere else.

**Comment by silbermm** [comment]
Now that we have unit tests, do we need to keep the ng2Test component around?

**Comment by silbermm** [comment]
Very nice work figuring out how to do dynamic content in NG2!

**Pull request merged**